easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -431,26 +432,26 @@ def determine_install_command(self):
 
             pip_verbose = self.cfg.get('pip_verbose', None)
             if pip_verbose or (pip_verbose is None and build_option('debug')):
-                self.cfg.update('installopts', '--verbose')
+                self.py_installopts.append('--verbose')
```

Comment:
Note that this is a behavior change: `self.cfg.update` only adds the value if it doesn't exist yet. Especially a double `--verbose` means "VERY verbose"